src/Qir/Controller/test-qir-controller.ps1

```diff
@@ -80,22 +82,21 @@ Foreach-Object {
         else {
             Write-Host "##[info]Test case '$($_.BaseName)' passed"
         }
-        continue;
-    }
-
-    $expectedError = Get-Content -Path $expectedErrorFile -Raw
-    $actualError = Get-Content -Path $errorFile -Raw
-    if (-not ($expectedError -ceq $actualError)) {
-        Write-Host "##vso[task.logissue type=error;]Failed QIR Controller test case: $($_.BaseName)"
-        Write-Host "##[info]Expected error:"
-        Write-Host $expectedError
-        Write-Host "##[info]Actual error:"
-        Write-Host $actualError
-        $script:all_ok = $False
-        continue
     }
     else {
-        Write-Host "##[info]Test case '$($_.BaseName)' passed"
+        $expectedError = Get-Content -Path $expectedErrorFile -Raw
```

Comment:
An error file can exist even when an output file exists. I think the right thing to do would be to treat the error file in a similar way than the output file. Check if a expected error file is present, if so, check if it is equal to the actual error file. If not, no error file should have been produced.